src/test/directives/onLongPress_test.ts

```diff
@@ -0,0 +1,89 @@
+import '../util.js';
+import {TestElement, sleep} from '../util.js';
+import {html} from 'lit';
+import {ref, createRef} from 'lit/directives/ref.js';
+import * as assert from 'uvu/assert';
+import * as hanbi from 'hanbi';
+import {onLongPress} from '../../main.js';
+
+suite('onLongPress directive', function () {
+  let element: TestElement;
+
+  this.timeout(5000);
```

Comment:
you can probably drop this by just choosing smaller timers

you can go slightly less realistic with the values, you're still testing it correctly. e.g. the happy path test can just use 10ms or something, small enough that the default timeout of tests will be fine